scripts/aws/deploy.py

```diff
@@ -126,6 +126,8 @@ def update_stage(client, rest_api_id, stage_name, stage_settings):
                         help="Location of edxapp for request routing")
     parser.add_argument("--catalog-host", required=True,
                         help="Location of catalog IDA for request routing")
+    parser.add_argument("--enterprise-host", required=True,
```

Comment:
One recommendation - if enterprise host is optional and is NOT set, consider setting it to the value passed into `edxapp_host` by default. We can override it as needed, and make it required after we're sure that it's in use everywhere.